src/sentry/api/endpoints/organization_events.py

```diff
@@ -338,6 +340,8 @@ def get(self, request: Request, organization) -> Response:
         elif referrer not in ALLOWED_EVENTS_REFERRERS:
             referrer = Referrer.API_ORGANIZATION_EVENTS.value
 
+        query_source = QuerySource.FRONTEND if is_frontend_request(request) else QuerySource.API
```

Comment:
Just a thought but it'd be nice if this was wrapped into a helper function like

```python
def get_request_source(request: Request) -> QuerySource:
  if is_frontend_request(request):
    return QuerySource.FRONTEND
  return QuerySource.API
```

This also gives you a single place to update in the future if there are additional query sources instead of having to update every caller with additional checks.